### PR TITLE
Add unspecified default value to all enums

### PR DESCRIPTION
**Pull request by theasianpianist**
Adds a `_UNSPECIFIED` value to all enums to act as the default value when a field is not set.

### CODE REVIEW

**Comment by Pterosaur** [comment]
Why do we need a value, _UNSPECIFIED ? Does it mean this enum is option?

**Comment by theasianpianist** [comment]
> Why do we need a value, _UNSPECIFIED ? Does it mean this enum is option?

Putting the _UNSPECIFIED value as the first item in the enum makes it the default value. E.g. if a `RoutingType` field is not set, when we try to get the value of that field it would return `ROUTING_TYPE_UNSPECIFIED` which we can check for and act appropriately. Previously, `ROUTING_TYPE_DIRECT` was the first item in this enum, so in the above scenario the field would return value `ROUTING_TYPE_DIRECT` which may not be correct.

**Review by Pterosaur — APPROVED** [review]
lgtm

File: proto/vnet_mapping.proto [hunk 1]
```diff
@@ -7,7 +7,7 @@ import "route_type.proto";
 
 message VnetMapping {
     // reference to routing type
-    route_type.RoutingType action_type = 1;
+    route_type.RoutingType routing_type = 1;
```
Comment (Pterosaur) [inline proto/vnet_mapping.proto:10]: I feel we should also add some PRs in sonic-swss and sonic-mgmt for this renaming.

Comment (theasianpianist) [reply proto/vnet_mapping.proto:10]: I will include the changes with my PL implementation PR for swss

**Pull request merged**